bedrock/firefox/templates/firefox/whatsnew/fx64/whatsnew-fx64-base.html

```diff
@@ -35,7 +35,7 @@
 {% set fxa_utm_params = {
   'campaign': 'wnp64',
```

Comment:
Do `campaign` and `content` here need updating?

Reply:
It looks like this template and the templates extending it aren't using the imported `fxa_email_form` macro so I'm going to remove it.
It also seems these values are being used by `fxa_link_fragment` which I'll update. Going to make a separate issue to standardize UTMs in other macros as well.